internal/services/containers/kubernetes_cluster_auth_resource_test.go

```diff
@@ -845,6 +848,9 @@ resource "azurerm_kubernetes_cluster" "test" {
     name       = "default"
     node_count = 1
     vm_size    = "Standard_DS2_v2"
+    upgrade_settings {
+      max_surge = "10%%"
```

Comment:
Can you also update the default value for this property for 4.0?

Reply:
The max_surge = 10% is only for the clusters created with k8s version = 1.28+, so I think we can't add the default value for this field.

Reply:
The window for supported versions will continue to move so at some point clusters can only be created with k8s version 1.28+. I don't see what is preventing us from having a default value in place for that field for v4.0.

Reply:
Is it correct `10%%` with double `%%` ?

Reply:
Yes, this syntax is required for `%` values in the test config